### ISSUE TITLE
When toggling before/after tiltak in the risk matrix, only show the new value

### ISSUE THREAD

**Issue by tarjeieo** [Puzzlepart/prosjektportalen#708]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Where in the Project Portal is this relevant?**

**Additional context**

**Is this something your organization is willing to contribute to financially?**
The Project Portal is a community-driven and dugnad-driven project, supported by organizations both in the public and private sector. If you're willing to contribute, please let us know (this will increase the chance of the feature being implemented). _Note: This is not a commitment, we will get back to you with an estimate before starting to work on the issue._

**Comment by olemp** [comment]
Can you add some more details and screenshots @tarjeieo?

**Comment by tarjeieo** [comment]
Should be OK if 0c3fa4e is correct
